README.md

```diff
@@ -56,6 +56,19 @@ npm install cron
 
 With the introduction of TypeScript in version 3 and alignment with UNIX cron patterns, a few changes have been made:
 
+<details>
+  <summary>Migrating from v3 to v4</summary>
```

Comment:
The header of this section is "Migrating from v2 to v3", it might benefit from being renamed to a more general name. The text right above this section "With the introduction of TypeScript ..." should be moved to the right section below. Or just create a new "Migrating from v3 to v4" section tbh, your call!